.github/workflows/visual-dev-storyshot.yml

```diff
@@ -0,0 +1,50 @@
+name: Visual Dev Storyshot Tests
```

Comment:
Doesn't this need to actually install a newer chromium version?

Reply:
Yes

https://coda.io/d/_d02Npt019Re/Storyshots-Setup-and-Usage_sutHZ#_lu__9

Reply:
Oh wait I misread your comment. I thought you were questioning whether it's necessary to install a newer Chromium.

The Chromium that comes with Puppeteer is outdated, so we just use the one that's bundled with the GH Action `ubuntu-latest` runner. I put a link to it alongside the Puppeteer environment variables config.